internal/runbits/cves/cves.go

```diff
@@ -40,7 +40,7 @@ func NewCveReport(prime primeable) *CveReport {
 }
 
 func (c *CveReport) Report(newBuildPlan *buildplan.BuildPlan, oldBuildPlan *buildplan.BuildPlan, names ...string) error {
-	changeset := newBuildPlan.DiffArtifacts(oldBuildPlan, false)
+	changeset := newBuildPlan.DiffArtifacts(oldBuildPlan, oldBuildPlan == nil)
```

Comment:
For cases where the old build plan is unavailable (e.g. `state checkout`, `state init`), we'll want to show only show CVEs for requested artifacts.